workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -157,8 +155,8 @@
   end
 
   describe '#artifacts_by_build_id' do
```

Comment:
Should updating space of variables for all 'it' of describe '#artifacts_by_build_id'

Reply:
I updated it